src/block/DeadBush.php

```diff
@@ -64,4 +64,14 @@ public function getFlameEncouragement() : int{
 	public function getFlammability() : int{
 		return 100;
 	}
+
+	private function isValidSupport(Block $block) : bool{
```

Comment:
This would be better named `canBeSupportedBy` for consistency with other blocks.